warehouse/macaroons/security_policy.py

```diff
@@ -66,7 +66,7 @@ def _extract_http_macaroon(request):
 
     if auth_method.lower() == "basic":
```

Comment:
just an idea. as I'm unsure how often we're using anything other than basic...

should we emit a lil metric here? just like `warehouse.macaroon.auth` with tag `auth_method:{auth_method.lower()` maybe done better as an entire PR instrumenting the session auth as well.